### ISSUE TITLE
MySQL extension install issues on self-hosted

### ISSUE THREAD

**Issue by oleg-brizy** [shivammathur/setup-php#220]
**Describe the bug**
I get warnings and mysql extension is not activated when I add this in workflow yaml

```yaml
with:
  uses: shivammathur/setup-php@v2
  extensions: ..., pdo_sqlite, mysql
```

```
PHP Warning:  PHP Startup: Unable to load dynamic library 'pdo_mysql.so'
 (tried: /usr/lib/php/20190902/pdo_mysql.so
 (/usr/lib/php/20190902/pdo_mysql.so: undefined symbol: pdo_parse_params),
 /usr/lib/php/20190902/pdo_mysql.so.so
 (/usr/lib/php/20190902/pdo_mysql.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
```

**To Reproduce**
1. self-hosted [`Dockerfile`](https://pastebin.com/0eTxzb3E)
2. My [`workflow.yml`](https://pastebin.com/Vk18uiyJ)

**Expected behavior**
It should install and activate the mysql extension without warnings.

**Screenshots**
https://imgur.com/lqPHUg1

**Additional context**
I was able to fix the problem by executing in container `apt install php7.4-mysql` and removing `pdo_sqlite, mysql` from workflow yaml

Log examples _(the warnings are multiplied on every new run)_
* [logs_52.zip](https://github.com/shivammathur/setup-php/files/4635464/logs_52.zip)
* [logs_53.zip](https://github.com/shivammathur/setup-php/files/4635465/logs_53.zip)
* [logs_54.zip](https://github.com/shivammathur/setup-php/files/4635467/logs_54.zip)

**Comment by shivammathur** [comment]
Fixed in 0158fe2. Please test in a new container.
If the issue is still there please provide logs by testing using `shivammathur/setup-php@verbose`.

**Comment by oleg-brizy** [comment]
It works. Thank you!
